packages/hint/src/lib/cli/analyze.ts

```diff
@@ -17,7 +17,6 @@ import {
     HintResources,
     UserConfig
 } from '../types';
-import { loadHintPackage } from '../utils/packages/load-hint-package';
```

Comment:
Same here, seems we could still use the old references. No need for the added diff.